app/api/user-remaning-msg/route.js

```
@@ -0,0 +1,27 @@
+export async function POST(req) {
+    const user = await currentUser();
+    const {token } = await req.json();
+    if(token){
+     const decision = await aj.protect(req, { userId:user?.primaryEmailAddress?.emailAddress, requested: token }); // Deduct 5 tokens from the bucket
+      if(decision.isDenied()){
+        return NextResponse.json({
+            error: 'Too many Resquest',
+            remainingToken: decision.reason.remaining
+        })
+
+        return NextResponse.json({allowed: true, remainingToken :decision.reason.remaining})
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)